ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/data/CustomerCenterState.kt

```diff
@@ -32,6 +32,7 @@ internal sealed class CustomerCenterState(
         @get:JvmSynthetic val promotionalOfferData: PromotionalOfferData? = null,
         @get:JvmSynthetic val title: String? = null,
         @get:JvmSynthetic override val navigationButtonType: NavigationButtonType = NavigationButtonType.CLOSE,
+        @get:JvmSynthetic val onNavigationOverride: (() -> Unit)? = null,
```

Comment:
Hmm so, IMO, if we're not using a navigation library, the VM should be in charge of holding the back stack. Then each Screen composable should use the "BackHandler" composable to customize the behavior upon a back button, then have the VM just update the state accordingly... I think having the state hold this is mixing the state with behavior, which should belong to the VM IMO.